.circleci/config.yml

```diff
@@ -4,23 +4,14 @@
 
 version: 2
 jobs:
-  build:
-    environment:
-      COVERAGE: true
+
+  checkout_code:
+    working_directory: ~/search-gov  
     docker:
       - image: circleci/ruby:2.3.5-browsers
         environment:
           RAILS_ENV: test
 
-      - image: circleci/mysql:5.6
-        environment:
-          MYSQL_ALLOW_EMPTY_PASSWORD: true
-          MYSQL_ROOT_HOST: "%"
-
-      - image: redis:3.2
-
-    working_directory: ~/search-gov
-
     steps:
```

Comment:
I noticed that sometimes `steps:` is followed by one line break, and sometimes two. For consistency, let's just use one line break throughout the file.

Reply:
@HalahAb , I think I was unclear. `steps:` should be followed by just one line break throughout the file, i.e.:
```
  steps:
    - restore_cache
```

Reply:
ah so sorry I misunderstood! I just pushed again!